consensus/hotstuff/verification/single_test.go

```diff
@@ -115,7 +115,8 @@ func TestSingleQC(t *testing.T) {
 
 	// verification with missing identity should be invalid
```

Comment:
```suggestion
	// verification with insufficient number of signers (no super-majority) should fail 
```

Reply:
I would say `(less than the random beacon threshold)` rather than `super-majority`.
Btw I find it a bit odd that `VerifyQC` and `CreateQC` only check the random beacon threshold and not the hotstuff threshold, which is done in the caller function.

Reply:
> odd that `VerifyQC` and `CreateQC` only check the random beacon threshold and not the hotstuff threshold

yea. that was the source of my confusion. Thanks for the clarification.

Reply:
the way you changed the comment is great. Thanks